<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPreorder</class>
 <widget class="QDialog" name="DlgPreorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tbl">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>UUID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Վիճակ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Գրանցման օր</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Կատարման օր</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Հաճախորդ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Հեռախոս</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Կանխիկ կանխավճար</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Քարտ կանխավճար</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ՀԴՄ</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnNew">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/add.png</normaloff>:/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnEdit">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/constructor.png</normaloff>:/constructor.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>740</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="chShowAll">
        <property name="text">
         <string>Ցուցադրել բոլորը</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnClose">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/res.qrc">
          <normaloff>:/logout.png</normaloff>:/logout.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="winfo" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Կանխավճար կանխիկով</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="C5DateEdit" name="leDateFor"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Կանխավճառ քարտով</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Գրանցման օր</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Հեռախոսի համար</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>Պահպանել</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QPushButton" name="btnCreateSale">
           <property name="text">
            <string>Կատարել վաճառք</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnPrintFiscal">
           <property name="text">
            <string>Տպել Կանխավճարի ՀԴՄ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="C5LineEdit" name="lePrepaidCart"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbState">
        <item>
         <property name="text">
          <string>Ընթացիկ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Կատարված</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Չեղարկված</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QTableWidget" name="tgoods">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>UUID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Goods id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Կոդ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Անվանում</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Քանակ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Գին</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ընդամենը</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnAddGoods">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/add.png</normaloff>:/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnEditGoods">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/constructor.png</normaloff>:/constructor.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>877</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Կատարման օր</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Հաճախորդ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5LineEdit" name="leUUID">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="C5LineEdit" name="lePhone"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>UUID</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="C5LineEdit" name="lePrepaidFiscal">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="C5LineEdit" name="leCustomer"/>
      </item>
      <item row="1" column="1">
       <widget class="C5DateEdit" name="leDateCreated"/>
      </item>
      <item row="7" column="1">
       <widget class="C5LineEdit" name="lePrepaidCash"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Վիճակ</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ՀԴՄ կտրոն</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
